### PR TITLE
915 TC-WalkerColina-04

### PR DESCRIPTION
**Pull request by wcolina**
**Test Case:**

image

**Automated Test Case Result:**

image

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/ExistingPPEList.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public class ExistingPPEList extends BasePage {
+
+    @FindBy(id = "btnNewExistingPpe")
+    private WebElement newExistingPpe;
+
+    @FindBy(className = "container")
+    private WebElement ppeContainer;
+
+    public ExistingPPEList() {
+        CommonEvents.isPresent(ppeContainer);
```
Comment (airmiky) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/ExistingPPEList.java]: Usar CommonEvents.isVisible(ppeContainer);

File: src/test/resources/features/ppe/ExistingPPEList.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# @Author: Walker Colina
+# feature to review behaviour of existing ppe list
+Feature: ExistingPPEList
+
+  Scenario: Una vez seleccionado el menu Existing PPE todos los Equipos existentes deben ser mostrados en una lista.
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/ExistingPPEList.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/ExistingPPEList.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    And Click 'Existing PPE' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/ExistingPPEList.feature]: Corregir a minuscula "click"

**Review by airmiky — COMMENTED** [review]
Revisar la observaciones por favor.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/ExistingPPEList.java (hunk 1 above)
Comment (wcolina) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/ExistingPPEList.java]: Done.

File: src/test/resources/features/ppe/ExistingPPEList.feature (hunk 2 above)
Comment (wcolina) [reply src/test/resources/features/ppe/ExistingPPEList.feature]: Done.

File: src/test/resources/features/ppe/ExistingPPEList.feature (hunk 3 above)
Comment (wcolina) [reply src/test/resources/features/ppe/ExistingPPEList.feature]: Done.

**Comment by wcolina** [comment]
All comments fixed. I merged the changes to issues in communication with team.

**Pull request merged**